File: 251120/최대로 겹치는 지점/maximum-overlapped-points.py
n = int(input())
segments = [tuple(map(int, input().split())) for _ in range(n)]

arr = [ 0 for _ in range(101) ]
for s, e in segments:
    for i in range(s, e + 1):
        arr[i] += 1


print(max(arr))

File: 260309/최대로 겹치는 지점/maximum-overlapped-points.js
const fs = require("fs");
const input = fs.readFileSync(0).toString().trim().split('\n');

const n = Number(input[0]);
const segments = [];
for (let i = 1; i <= n; i++) {
  segments.push(input[i].split(' ').map(Number));
}

let answer = Array(101).fill(0);

for (let [x1, x2] of segments) {
  for (let i = x1; i <= x2; i++) {
    answer[i] += 1;
  }
}

console.log(Math.max(...answer))